### PR TITLE
Update advice for getting support and filing issues.

### PR DESCRIPTION
**Pull request by bgrant0607**
Re. discussion about directing people to stackoverflow for support.

CONTRIBUTING.md is what users are directed to from the new issue page. Happy to add more details, since we don't have an issue template yet.

cc @quinton-hoole @brendandburns @dchen1107

### CODE REVIEW

File: docs/troubleshooting.md [hunk 1]
```diff
@@ -56,19 +56,25 @@ You may also find the StackOverflow topics relevant
...
-If you have what looks like a bug, or you would like to make a feature request, please use the [Github issue tracking system](https://github.com/GoogleCloudPlatform/kubernetes/issues).
+If you have what looks like a bug, or you would like to make a feature request, please use the [Github issue tracking system](https://github.com/kubernetes/kubernetes/issues).
 
 Before you file an issue, please search existing issues to see if your issue is already covered.
 
+If filing a bug, please include information about how to reproduce the problem.
+
 # Help! My question isn't covered!  I need help now!
 
+## Stackoverflow
```
Comment (quinton-hoole) [inline docs/troubleshooting.md]: I would suggest re-ordering this troubleshooting guide to direct new users first to Stackoverflow for troubleshooting help, and only after they have exhausted that option, consider filing bug reports or feature requests.  From my experience things that look like bugs or missing features to new users often evaporate once more experienced users offer alternative approaches or configuration debugging support.

**Comment by quinton-hoole** [comment]
One nit, otherwise LGTM.

**Comment by quinton-hoole** [comment]
I'm not sure why github thinks I closed this - I did not intend to.  Nor am I apparently able to re-open.

**Comment by bgrant0607** [comment]
Thanks. Comments addressed. PTAL. I'll squash after lgtm.

**Comment by thockin** [comment]
lgtm

**Comment by bgrant0607** [comment]
Thanks. Squashed.

**Pull request merged**